packages/website/data/news/hello-world/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Hello World
+date: "2015-05-01T22:12:03.284Z"
+description: "Hello World"
+author: Nicole Tay
+avatar: ../../team/nicole-tay.png
+hidden: true
+---
+
+This is my first post on my new fake blog! How exciting!
+
+I'm sure I'll write a lot more interesting things in the future.
+
+Oh, and here's a great quote from this Wikipedia on
+[salted duck eggs](https://en.wikipedia.org/wiki/Salted_duck_egg).
+
+> A salted duck egg is a Chinese preserved food product made by soaking duck
+> eggs in brine, or packing each egg in damp, salted charcoal. In Asian
+> supermarkets, these eggs are sometimes sold covered in a thick layer of salted
+> charcoal paste. The eggs may also be sold with the salted paste removed,
+> wrapped in plastic, and vacuum packed. From the salt curing process, the
+> salted duck eggs have a briny aroma, a gelatin-like egg white and a
+> firm-textured, round yolk that is bright orange-red in color.
+
+![Chinese Salty Egg](./salty_egg.jpg)
+
```

Comment:
TBH,  didn't manage to find where this text is being used/displayed. But do we need it? Until here it looks OK but then it turns into a bunch of lorem ipsum.

Reply:
haha yeah it's hidden on the website :) it's just there for reference or an example